web/js-file.js

```diff
@@ -1,6 +1,8 @@
 function funk(a,b) {
  if (a == b)
   return a;
+ if (a == d)
+  return a;
  if (a == null)
   return b;
  if (c)
```

Comment:
Reporter: JSHINT
Rule: ?
Severity: WARN
File: ../../../web/js-file.js L8
 

&apos;c&apos; is not defined.: if (c)

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>